lib/sorcery/controller/config.rb

```diff
@@ -12,6 +12,8 @@ class << self
         attr_accessor :save_return_to_url
         # set domain option for cookies
         attr_accessor :cookie_domain
+        # Set token randomness
+        attr_accessor :token_randomness
```

Comment:
`Sorcery:Controler::Config` is configuration used for `Sorcery::Controller`. But `token_randomness` is used for `Sorcery::Model::TemporaryToken` So, It is better to set it with `Sorcery:Model::Config` like [encryption_algorithm](  https://github.com/Sorcery/sorcery/blob/4e3fcdb8df16912424281322ce9f2862937846fc/lib/sorcery/model/config.rb#L44)